src/PaperG/ScenarioBlob/Blobs/UnmanagedFacebook/FacebookAdSet.php

```diff
@@ -166,4 +166,4 @@ private function validateGoal($validOptimizationGoals)
 
         return in_array($this->optimizationGoal, $validOptimizationGoals);
     }
-} 
+}
```

Comment:
It does belong in the Facebook scenario, but it is in both managed and unmanaged scenarios.  And since we haven't fully transitioned facebook, technically it is part of the `FirehoundBlob` scenario still.

I also favor a minimal moving of classes, because the PR in firehound to use this version of the library will be kind of a pain.